lib/cryptoexchange/exchanges/eosex/services/market.rb

```diff
@@ -0,0 +1,48 @@
+module Cryptoexchange::Exchanges
+  module Eosex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Eosex::Market::API_URL}"
+        end
+
+        def adapt_all(output)
+          output['msg'].map do |pair|
+            base, target = pair["Symbol"].split('/')
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+              base: base,
+              target: target,
+              market: Eosex::Market::NAME
+            )
+            adapt(pair, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Eosex::Market::NAME
+          ticker.last      = NumericHelper.to_d(output['LastPrice'])
+          ticker.high      = NumericHelper.to_d(output['High'])
+          ticker.low       = NumericHelper.to_d(output['Low'])
+          ticker.volume    = NumericHelper.to_d(output['BaseVolume'])
```

Comment:
The volume in our API needs to refer to the base currency https://github.com/coingecko/cryptoexchange/wiki/Implementing-a-New-Exchange-(HTTP-API)#some-things-to-take-note

For example here EOS/ETH
```

MarketId | 2002
-- | --
Name | "EOSETH"
Symbol | "EOS/ETH"
Updated | 1549955400
LastPrice | "0.02291400"
Low | "0.02259500"
High | "0.02821200"
Open | "0.02273000"
Close | "0.02291400"
BaseVolume | "372.81125413"
BaseVolumeBuy | "233.13746257"
BaseVolumeSell | "139.67379156"
```
Visiting https://eosex.com/#/exchange/EOSETH it will saye 372 ETH for 24h volume

You'll need to obtain volume in EOS by dividing the price. There are existing examples implemented in other exchange